README.md

```diff
@@ -4,9 +4,25 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
-```
-npm install @smg-automotive/example
-```
+
+1. Install the package
+   ```sh
+   npm install @smg-automotive/components
```

Comment:
Thank you for adding the instructions.
Do we need to highlight that the `^` needs to be removed.

Reply:
I wouldn't. Modern npm shouldn't add them by default if all other dependencies are pinned. Worst case we ger renovate PR to pin